README.md

```diff
@@ -63,20 +63,21 @@ git checkout 1.2
 Note that release branches might contain non-released patches.
 
 ## GHC compatibility
-|      | Linux | Windows | macOS |
-|------|-------|---------|-------|
-| 8.6  | ✔️     | ✔️      | ✔️     |
-| 8.8  | ✔️     | ❌      | ✔️     |
-| 8.10 | ✔️     | ✔️      | ❌     |
-| 9.0  | ✔️     | ✔️²     | ✔️     |
-| 9.2¹ | ❌     | ❌      | ❌     |
-| 9.4³ | ❌     | ❌      | ❌     |
+|      | Linux | Windows | macOS | Clash (released) | Clash (development version)
+|------|-------|---------|-------|------------------|--------------------------
+| 8.6  | ✔️     | ✔️       | ✔️     | 1.6              | ✔️
+| 8.8  | ✔️     | ❌      | ✔️     | 1.6              | ✔️
+| 8.10 | ✔️     | ✔️       | ❌    | 1.6              | ✔️
+| 9.0  | ✔️     | ✔️²      | ✔️     | 1.6              | ✔️
```

Comment:
```suggestion
| 8.6  | ✔️     | ✔️       | ✔️     | 1.0              | ✔️
| 8.8  | ✔️     | ❌      | ✔️     | 1.0              | ✔️
| 8.10 | ✔️     | ✔️       | ❌    | 1.2              | ✔️
| 9.0  | ✔️     | ✔️²      | ✔️     | 1.4              | ✔️
```

Reply:
I've updated the suggestion to include the range as we discussed. Is this the version you'd like to see?

Reply:
Yes looking good